pylibs/unittests/test_basic.py

```diff
@@ -965,6 +965,73 @@ def testPhyStats(self):
         self.assertTrue(os.path.isfile('tmp/0_txbytes.csv'))
         self.assertTrue(os.path.isfile('tmp/0_chansamples.csv'))
 
+    def testBackgroundCommands(self):
+        ns: OTNS = self.ns
+        ns.radiomodel = 'MIDisc'
+
+        nid_ldr = ns.add("router", x=600, y=300)  # Leader
+        ns.node_cmd(nid_ldr, "prefix add 2001:dead:beef:cafe::/64 paros med")
+        ns.node_cmd(nid_ldr, "netdata register")
+        ns.go(10)
+
+        nid_srv = ns.add("router", x=600, y=500)  # Router/REED aka 'server'
+        ns.go(10)
+
+        # Add Children to the Router
+        ns.add("med")
+        ns.add("sed")
+        ns.add("sed")
+        ns.add("sed")
+        ns.add("fed")
+        ns.go(10)
+
+        # Add a neighbour Router
+        nid_diag = ns.add("router", x=670, y=320)
+        ns.go(130)
+
+        a_rloc16 = ns.get_rloc16(nid_srv)
+        a_rloc = ns.get_ipaddrs(nid_srv, 'rloc')[0]
+        a_mleid = ns.get_ipaddrs(nid_srv, 'mleid')[0]
+
+        # Execute several 'background' commands. These will not return an immediate 'Done' CLI response, but will
+        # run in the background. OTNS will return a 'Started' CLI response to signal the command has been started.
+        # The 'Done' response will follow asynchronously.
+        ns.node_cmd(nid_ldr, f'ping {a_mleid}')
+        ns.go(10)
+
+        # The 'async' invocation turns the command into a foreground command again. However this case should also
```

Comment:
The addition of `async` turns the command into a foreground command?

Reply:
Maybe the formulation was unclear - I've now updated this test part with some extra assert checks and more explanation in the comment.

The "background" term for OTNS equals "foreground" in the OT CLI i.e. in an actual OT CLI the command would block. And with "async" the ping command does not block anymore.
So also "foreground" in OTNS equals "background" in an OT CLI used without simulator.

Now quotes are used to remind a reader that 'background' is not equal to background what a person used to the OT CLI may know as a term.

It's possible also to create new terminology for this, if needed.